articles/how-to-make-an-image-carousel-for-your-website/index.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/how-to-make-an-image-carousel-for-your-website/
+title: How to make an Image Carousel for your website using HTML,CSS and JS
+description: This article takes the reader in a step by step process on how to make an image carousel and make their web pages more interactive and lively to the user hence enhancing their experience.
+author: eugiene-kanillar
+date: 2021-03-15T00:00:00-22:00
+topics: []
+excerpt_seperator: <!--moore-->
+images:
+
+ -url: /engineering-education/how-to-make-an-image-carousel-for-your-website/hero.PNG
+  alt: Sample image of the Carousel
+---
+An Image Carousel is like a container of Images which the user can select through by clicking a buttons which directs him forward or backwards in the collection of images. It enables the website to be more interactive and hence enhance the user experience. The collection of Images can be set to change automatically by use of a timer or manually when the user clicks the displayed buttons.
+<!--moore-->
+# Goal
```

Comment:
Do note that we use h3 for all our headings as highlighted in the pull request requirement. So kindly update all your heading to h3.